index.html

```diff
@@ -48,30 +51,182 @@ <h2>I'm a Software Developer</h2>
+      </section>
+      <section class="projthirdpage">
+        <div id="project5">
+          <h2 class="work-subtitle protitle">The Quiz</h2>
+          <p class="work-details prodetails">Make a quiz templete using html forms</p>
+          <div class="flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+        <div id="project6">
+          <h2 class="work-subtitle protitle pro2title">Tribute Page</h2>
+          <p class="work-details prodetails">Tribute to Dr. Abdul Salam a Nobel Scientist.</p>
+          <div class="flex pro2flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+      </section>
+      <section id="aboutme">
+        <h1 id="me">About me</h1>
+        <p>
+          I can help you to build a product, feature or website Look through
+          some of my work and experience! If you like what you see and have a
+          project you need coded, don't hesitate to contact me.
+        </p>
+        <button class="work-button resumebtn">Get My Resume</button>
+        <div><img id="aboutmeimg" src="/img/illustration-about-me-mobile.png" alt=""></div>
+        </section>
+        <section id="languages">
+          <div class="thumbnail">
+          <div class="rectangle">
+          </div>
+          <h1 class="headings">Languages</h1>
+        </div>
+          <ul class="marginlistitems">
+            <li class="listitems">Javascript</li>
+            <li class="listitems">Ruby</li>
+            <li class="listitems">Html</li>
+            <li class="listitems">Css</li>
+          </ul>
+          <div class="thumbnail">
+            <div class="rectangle square">
+            </div>
+            <h1 class="headings">Frameworks</h1>
+          </div>
+          <ul class="marginlistitems">
+            <li class="listitems">Bootstrap</li>
+            <li class="listitems">Ruby on Rails</li>
+            <li class="listitems">RSPec</li>
+            <li class="listitems">CapyBara</li>
+            <li class="listitems">Selenium</li>
+          </ul>
+          <div class="thumbnail">
+            <div class="rectangle circle">
+            </div>
+            <h1 class="headings ">Skills</h1>
+          </div>
+          <ul class="marginlistitems">
+            <li class="listitems ">Codekit</li>
+            <li class="listitems">Github</li>
+            <li class="listitems">Codepen</li>
+            <li class="listitems">Gitlab</li>
+            <li class="listitems">Terminal</li>
+          </ul>
+        </section>
+        <section id="getintouch" class="border">
```

Comment:
- [ ] I would implore you to give space between these tags because they are overlapping.

### Your work
![image](https://user-images.githubusercontent.com/86200356/219904012-fa9875f7-b5c3-4981-98b0-d5ba0a21ff40.png)

### Figma design
![image](https://user-images.githubusercontent.com/86200356/219904049-a1484cbc-af4e-4c0a-98e4-3a86c98beed5.png)

Reply:
![image](https://user-images.githubusercontent.com/112868013/219905586-e70ccd7a-71bd-4e73-a395-97cccaa07613.png)
@Olamarx 
it looks like this at my side please review it again. or send me your whatsapp number so I share video of this project to you. please be kind and review it again.